assets/src/dashboard/constants/stories.js

```diff
@@ -107,9 +107,21 @@ export const STORY_ITEM_CENTER_ACTION_LABELS = {
 };
 
 export const STORY_STATUSES = [
-  { label: __('All Stories', 'web-stories'), value: STORY_STATUS.ALL },
-  { label: __('Drafts', 'web-stories'), value: STORY_STATUS.DRAFT },
-  { label: __('Published', 'web-stories'), value: STORY_STATUS.PUBLISHED },
+  {
+    label: __('All Stories', 'web-stories'),
+    value: STORY_STATUS.ALL,
+    status: 'all',
```

Comment:
Open to better ideas for this. I was trying to stay in scope. the current "all" status is "publish,draft" for the sake of the API. I feel like changing it at the api level to have a `STORY_QUERY_STATUS[STORY_STATUS.value]` would be useful, but I'm on a constants train right now and don't want to affect something inadvertently since we are still waiting to see were data lands.

Reply:
My guess is the API uses the comma separated values of `publish,draft` to filter where `all` would require some additional logic. I think doing this in the client is fine.

Reply:
yeah that's exactly what the status is getting used for - i'll leave this like this for the time being then.